src/cloud_provider/scaleway/kubernetes/mod.rs

```diff
@@ -578,11 +580,195 @@ impl<'a> Kapsule<'a> {
     }
 
     fn pause(&self) -> Result<(), EngineError> {
-        todo!()
+        let listeners_helper = ListenersHelper::new(&self.listeners);
+        let send_to_customer = |message: &str| {
+            listeners_helper.pause_in_progress(ProgressInfo::new(
+                ProgressScope::Infrastructure {
+                    execution_id: self.context.execution_id().to_string(),
+                },
+                ProgressLevel::Info,
+                Some(message),
+                self.context.execution_id(),
+            ))
+        };
+        send_to_customer(format!("Preparing SCW {} cluster pause with id {}", self.name(), self.id()).as_str());
```

Comment:
there is now a self.send_to_customer method

Reply:
Done

Reply:
Changed in AWS and DO as well